includes/api/class-wc-admin-rest-products-controller.php

```diff
@@ -127,7 +127,7 @@ public static function add_wp_query_filter( $where, $wp_query ) {
 		}
 
 		if ( $wp_query->get( 'low_in_stock' ) ) {
-			$low_stock_amount = get_option( 'woocommerce_notify_low_stock_amount', 2 );
+			$low_stock_amount = absint( max( get_option( 'woocommerce_notify_low_stock_amount', 2 ) ) );
```

Comment:
`max()` is unnecessary here and will throw a warning if the first argument isn't an array.

```
php > var_dump( max( 'a' ) );
Warning: max(): When only one parameter is given, it must be an array
```

Reply:
@jeffstieler Changed this back to not use `max()`, but do you know why they do this throughout core?  I assumed there was some plugin that used an array as most instances are written as:

```php
absint( max( get_option( 'woocommerce_notify_low_stock_amount', 2 ) ) );
```

Reply:
@joshuatf the core uses of `max()` provide a second argument: https://github.com/woocommerce/woocommerce/blob/f9cdab66eb1f61f6f26001838199aa1a5b8ba01f/includes/admin/reports/class-wc-report-low-in-stock.php#L39

From https://www.php.net/max:
> If the first and only parameter is an array, max() returns the highest value in that array. If at least two parameters are provided, max() returns the biggest of these values.

Reply:
🤦‍♂️ I kept reading this as the second parameter that acts as the default value in `get_option()`.  Going to swap this for a max with `1` since this seems to be the most commonly used in core for low stock reporting.